framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -3232,12 +3239,43 @@ VkResult VulkanReplayConsumerBase::OverrideAllocateMemory(
                 }
                 break;
             }
+
+            if (current_struct->sType == VK_STRUCTURE_TYPE_IMPORT_MEMORY_HOST_POINTER_INFO_EXT)
+            {
+                auto import_info = reinterpret_cast<VkImportMemoryHostPointerInfoEXT*>(current_struct);
+
+                host_pointer_size = util::platform::GetAlignedSize(replay_allocate_info->allocationSize,
```

Comment:
Why does the size have to rounded to multiples of the system's page size?

Reply:
Hi @panos-lunarg 

It was implemented in the same way inside `VulkanCaptureManager::OverrideAllocateMemory` function 3 years ago
With comment explaining why:
```
// TODO: This should be aligned to minImportedHostPointerAlignment, but there is a currently a loader bug
// that prevents the layer from querying for that value when a 1.0 application does not explicitly enable
// physical_device_properties2.  For now we align to system page size.
```

Looks stable for now, but if you don't mind I can reimplement with minImportedHostPointerAlignment rounding, based on VK-GL-CTS examples. VK-GL-CTS uses `posix_memalign` api for that

Reply:
I see. makes sense